api-test/src/main/resources/pms/ABISPartnerV3/UploadCertificateABIS/UploadCertificateABISForCASUBCA.yml

```
@@ -0,0 +1,29 @@
+UploadCertificateABISCASUBCA:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)